cylc/uiserver/handlers.py

```diff
@@ -45,45 +49,54 @@ def set_default_headers(self) -> None:
         self.set_header("Content-Type", 'application/json')
 
 
-class MainHandler(HubOAuthenticated, BaseHandler):
+class MainHandler(HubOAuthenticated, BaseHandler):  # type: ignore
```

Comment:
The only places where I used the `type: ignore` flag to disable mypy were places where subclassing a type without type hints was necessary, like here where we subclass types from JupyterHub that doesn't have type stubs yet.